storybook/src/components/Heading/Heading.docs.mdx

```diff
@@ -14,21 +14,22 @@ import README from "../../../../packages/css/src/components/heading/README.md?ra
 
 ## Examples
 
-### Heading 1
+### Levels
 
-<Canvas of={HeadingStories.Heading1} />
+Four levels of headings are available.
+A page should contain only one Heading with level 1.
+Do not skip heading levels.
+User agents can use heading information to construct a table of contents for a document.
```

Comment:
Shouldn't this be in the guidelines section? (Most of it is already)

Reply:
Yes, moved up. And rewritten most of the guidelines.